kodilla-hibernate/src/test/java/com/kodilla/hibernate/tasklist/dao/TaskListDaoTestSuite.java

```diff
@@ -29,13 +29,13 @@ public void testFindByListName() {
         //When
         List<TaskList> readTaskList = taskListDao.findByListName(name);
```

Comment:
Tutaj powinnaś użyć od razu `LISTNAME` zamiast `name`. Jeśli do sekcji when wykorzystasz nazwę zwrówoną z zapisanego obiektu, to jeśli byłaby inna niż zdefiniowana przez Ciebie - będziesz szukać niewłaściwego rekordu.